app/uk/gov/hmrc/exports/controllers/SubmissionController.scala

```diff
@@ -34,7 +34,7 @@ class SubmissionController @Inject()(
 )(implicit executionContext: ExecutionContext)
     extends RESTController(cc) with JSONResponses {
 
-  def create(id: String): Action[Unit] = authenticator.authorisedAction(parse.empty) { implicit request =>
+  def create(id: String): Action[AnyContent] = authenticator.authorisedAction(parse.default) { implicit request =>
```

Comment:
Out of curiosity, why you changed this to `AnyContent`?

Reply:
Firstly - I noticed it makes difference for testing and you can't use the same patterns.
Secondly - I don't see any benefit from having `Action[Unit]` here. So it was easier for me to do it this way, than to figure out a new way for testing.